tedium.js

```diff
@@ -27,7 +29,41 @@ const path = require('path');
 const ProgressBar = require('progress');
 const rimraf = require('rimraf');
 const cleanup = require('./cleanup-passes');
-
+const hydrolysis = require('hydrolysis');
+const pad = require('pad');
+const cliArgs = require("command-line-args");
+const cli = cliArgs([
+  {
+    name: "help",
+    type: Boolean,
+    alias: "h",
+    description: "Print usage."
+  },
+  {
+    name: "max_changes",
```

Comment:
Development. When we run tedium in cron it'll be with --max_changes=100 or similar, but as I was hacking on this code I wanted it to fail closed and not send out a ton of bad changes.

On reflection, this wasn't super useful by itself, so I just added another method that also reports on the repos that would have been pushed up, so that you can examine the changes yourself.

Also commented all of the functions (after you've already had to read them and wrestle out what they did.. oops)